test/cpp/end2end/end2end_test.cc

```diff
@@ -1381,6 +1381,77 @@ TEST_P(End2endTest, ExpectErrorTest) {
   }
 }
 
+TEST_P(End2endTest, DelayedRpcEarlyCanceledUsingCancelCallback) {
+  MAYBE_SKIP_TEST;
+  if (!GetParam().callback_server || GetParam().use_interceptors) {
```

Comment:
I'd suggest either copying the comment below up here.

Reply:
Done.